mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,156 @@
+= Listen for New Messages with MQTT Connector
+
+In these examples, you configure the Anypoint Connector for MQTT (MQTT Connector) *On New Message* source that enables you to listen for new incoming messages using one or more topic filters. Each topic filter has a specific quality of service (QoS) configured.
+
+In the following example, you configure the *On New Message* source:
+
+. In the *Mule Palette* view of Studio, select *MQTT3 > On New Message*.
```

Comment:
GLOBAL type of rewording alternative:
```suggestion
. In Studio > *Mule Palette*, select *MQTT3 > On New Message*.
```
I'll stop making these suggestions about replacing verbiage with ">", just so I don't drive you crazy!